### PR TITLE
Add VTheme template

### PR DESCRIPTION
**Pull request by MikeZeDev**
and update existing websites to use it

### CODE REVIEW

**Comment by ronny1982** [comment]
Consider adding other websites:
- https://philiascans.org

File: web/src/engine/websites/templates/Iken.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    number: number,
+    title: string
+}
+
+type MangaID = {
+    id: string,
+    slug: string,
+}
+
+const pageScript = `[...document.querySelectorAll('section img[loading]')].map(image => new URL(image.getAttribute('src'), window.location.origin).href);`;
```
Comment (ronny1982) [inline web/src/engine/websites/templates/Iken.ts:32]: Can be simplified by using property instead of attribute `[...].map(img => img.src);`

**Review by ronny1982 — APPROVED** [review]

**Comment by ronny1982** [comment]
Paid chapters are not filtered, is this by intention?

File: web/src/engine/websites/templates/Iken.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+@Common.PagesSinglePageJS(pageScript, 2500)
+@Common.ImageAjax()
+export class Iken extends DecoratableMangaScraper {
```
[the remark this answers is not in the document]
Comment (MikeZeDev) [reply web/src/engine/websites/templates/Iken.ts:36]: Got it

**Comment by MikeZeDev** [comment]
> Paid chapters are not filtered, is this by intention?

Generally, if the api doesnt give a way to distinguish paid chapters i'd rather not rely on css to filter content. I am gonna double check here to be sure, memory issues.

If you want me to follow a particular guideline for paid content, just tell me.

**Pull request merged**